README.md

```diff
@@ -107,15 +107,22 @@ app.get('/login',
 
 ## API access
 
-If you want to get a list of connections or users from auth0, use the [auth0 module](https://github.com/auth0/node-auth0).
+If you want to get a list of connections or users from Auth0, use the [Auth0 module](https://github.com/auth0/node-auth0).
 
 ## Complete example
 
-A complete example of using this library [here](http://github.com/auth0/passport-auth0).
+A complete example of using this library is shown [here](http://github.com/auth0/passport-auth0).
 
 ## Documentation
 
-For more information about [auth0](http://auth0.com) contact our [documentation page](http://docs.auth0.com/).
+For more information about [Auth0](http://auth0.com) contact our [documentation page](http://docs.auth0.com/).
```

Comment:
> ... contact our documentation page

Funny way of putting that :) 

> http://docs.auth0.com/

Wrong URL 

I wonder if we shouldn't point them somewhere else? Docs does not have anything about Passport that I know of.

Reply:
@joshcanhelp Yeah, that part was reading wrong to me. I think here, we could rather prompt them to sign up.

What about something like this:

## Auth0: Never Compromise on Identity

At [Auth0](https://auth0.com/), we make heavy use of full-stack JavaScript to help our customers [manage user identities which includes password resets along with creating, provisioning, blocking, and deleting users](https://auth0.com/user-management). Therefore, it must come as no surprise that using our identity management platform on JavaScript web apps is a piece of cake.

[Auth0 offers a **free tier**](https://auth0.com/pricing) to get you started with modern authentication. Check it out, or <a href="https://auth0.com/signup" data-amp-replace="CLIENT_ID" data-amp-addparams="anonId=CLIENT_ID(cid-scope-cookie-fallback-name)">sign up for a free Auth0 account here</a>!

![Auth0 Login Page](https://cdn2.auth0.com/docs/media/articles/web/hosted-login.png)

You may also check out our [Quick Start tutorials](https://auth0.com/docs/quickstarts) to learn how to implement authentication using different languages and frameworks in your apps.

Reply:
Here's the standard blurb we use on our SDKs:

https://github.com/auth0/open-source-template/blob/master/README-sample.md#what-is-auth0

That's content-approved so should be fine to use anywhere!